packages/expect/src/jasmineUtils.ts

```diff
@@ -238,19 +237,19 @@ function keys(obj, isArray, hasKey) {
   return extraKeys;
 }
 
-function hasDefinedKey(obj, key) {
+function hasDefinedKey(obj: any, key: string) {
   return hasKey(obj, key) && obj[key] !== undefined;
 }
 
-function hasKey(obj, key) {
+function hasKey(obj: any, key: string) {
   return Object.prototype.hasOwnProperty.call(obj, key);
 }
 
-export function isA(typeName: string, value: any) {
+export function isA(typeName: string, value: unknown) {
   return Object.prototype.toString.apply(value) === '[object ' + typeName + ']';
 }
 
-function isDomNode(obj) {
+function isDomNode(obj: any) {
```

Comment:
You could type gate here ie,

```ts
- function isDomNode(obj: any) {
+ function isDomNode(obj: any): obj is Node {
```

helps with things like:
```ts
const element: any = findSomething():

if (isDomNode(element)) {
   element.appendChild // will now intellisense 
}
```

Reply:
Oh cool. I didnt't know about type guards. Looks reasonable